.circleci/flake8_config.ini

```diff
@@ -5,60 +5,113 @@ max-line-length = 80
 ignore =
   # Black conflicts and overlaps.
   # Found in https://github.com/psf/black/issues/429
-  B950, # Line too long. (Use `arc lint`'s LINEWRAP instead)
-  E111, # Indentation is not a multiple of four.
-  E115, # Expected an indented block (comment).
-  E117, # Over-indented.
-  E121, # Continuation line under-indented for hanging indent.
-  E122, # Continuation line missing indentation or outdented.
-  E123, # Closing bracket does not match indentation of opening bracket's line.
-  E124, # Closing bracket does not match visual indentation.
-  E125, # Continuation line with same indent as next logical line.
-  E126, # Continuation line over-indented for hanging indent.
-  E127, # Continuation line over-indented for visual indent.
-  E128, # Continuation line under-indented for visual indent.
-  E129, # Visually indented line with same indent as next logical line.
-  E201, # Whitespace after '('.
-  E202, # Whitespace before ')'.
-  E203, # Whitespace before ':'.
-  E221, # Multiple spaces before operator.
-  E222, # Multiple spaces after operator.
-  E225, # Missing whitespace around operator.
-  E226, # Missing whitespace around arithmetic operator.
-  E227, # Missing whitespace around bitwise or shift operator.
-  E231, # Missing whitespace after ',', ';', or ':'.
-  E241, # Multiple spaces after ','.
-  E251, # Unexpected spaces around keyword / parameter equals.
-  E261, # At least two spaces before inline comment.
-  E262, # Inline comment should start with '# '.
-  E265, # Block comment should start with '# '.
-  E271, # Multiple spaces after keyword.
-  E272, # Multiple spaces before keyword.
-  E301, # Expected 1 blank line, found 0.
-  E302, # Expected 2 blank lines, found 0.
-  E303, # Too many blank lines (3).
-  E305, # Expected 2 blank lines after end of function or class.
-  E306, # Expected 1 blank line before a nested definition.
-  E501, # Line too long (82 > 79 characters).
-  E502, # The backslash is redundant between brackets.
-  E701, # Multiple statements on one line (colon).
-  E702, # Multiple statements on one line (semicolon).
-  E703, # Statement ends with a semicolon.
-  E704, # Multiple statements on one line (def).
-  W291, # Trailing whitespace.
-  W292, # No newline at end of file.
-  W293, # Blank line contains whitespace.
-  W391, # Blank line at end of file.
+  # B950: Line too long. (Use `arc lint`'s LINEWRAP instead)
```

Comment:
Are these changes intentional?

Reply:
Yes (I've actually had to learn a new PyCharm shortcut to make it happen).
Flake8 config no [longer support](https://flake8.pycqa.org/en/latest/user/configuration.html) inline comments, so the fix is due

I've updated the PR description too.